tq_website/urls.py

```diff
@@ -49,6 +49,7 @@
     path("accounts/", include("allauth.urls")),
     path("password/", courses_views.change_password, name="change_password"),
     path("profile/courses", courses_views.user_courses, name="user_courses"),
+    path("profile/<int:user_id>/courses.ical", courses_views.user_courses_ical, name="user_courses_ical"),
```

Comment:
```suggestion
    path("profile/<int:user_id>/calendar.ical", courses_views.user_ical, name="user_ical"),
```

More generic URL, in case we add events add some point